CONTRIBUTING.md

```diff
@@ -125,6 +139,35 @@ web        | [BE] NEW_RELIC_APP_NAME. Not starting!
 
 You do not need to set up NewRelic, except if you fix anything related to that.
 
+##### MySQL error during startup
+
+During the server start (after running ‘yarn start’), you might notice an error log similar to this:
+
+```
+at Class.exports.up (/Users/admin/Desktop/myprojects/mozilla/voice-web-master/server/src/lib/model/db/migrations/20180910121256-user-sso-fields.ts:2:13)
+....
+[BE]       at /Users/admin/Desktop/myprojects/mozilla/voice-web-master/node_modules/db-migrate/lib/migrator.js:237:31 {
+[BE]     code: 'ER_BLOB_CANT_HAVE_DEFAULT',
+[BE]     errno: 1101,
+[BE]     sqlMessage: "BLOB, TEXT, GEOMETRY or JSON column 'username' can't have a default value",
+[BE]     sqlState: '42000',
+[BE]     index: 0,
+[BE]     sql: '\n' +
+[BE]       '      ALTER TABLE user_clients\n' +
+[BE]       '        ADD COLUMN sso_id VARCHAR(255) UNIQUE,\n' +
+[BE]       "        ADD COLUMN username TEXT NOT NULL DEFAULT '',\n" +
+[BE]       '        ADD COLUMN basket_token TEXT;\n' +
+[BE]       '    '
+```
+
+First make sure you have the correct version of MySQL (v8.0 or greater) installed. If the problem still persists, the following will prove useful:
+**You only need to follow these steps once. After that be sure to discard all changes made to the relevant files after migrations are successful.**
```

Comment:
Slight nitpick to avoid making people do unnecessary install steps - we're actually using MySQL 5.7 on the server, haha, and the `ER_BLOB_CANT_HAVE_DEFAULT` issue is actually something that even 5.x versions enforce: https://dev.mysql.com/doc/refman/5.7/en/data-type-defaults.html

The other way to allow migration to pass through is to turn off in your local, which I didn't bother trying to figure out because we were screensharing: 

>  If strict mode is not enabled, MySQL sets the column to the implicit default value for the column data type. 

Your stated workaround here is still fine, just remove the bit about server versioning.

Reply:
ha okay got it.